docs/language/crypto.mdx

```diff
@@ -31,6 +31,12 @@ pub enum HashAlgorithm: UInt8 {
     /// This is a customized version of KMAC128 that is compatible with the hashing to curve 
     /// used in BLS signatures.
     pub case KMAC128_BLS_BLS12_381 = 5
+
+    /// Returns the hash of the given data
+    pub fun hash(_ data: [UInt8]): [UInt8]
```

Comment:
I think we can update the doc by taking the hashing section out of the crypto contract section. 
We could also update the example [here](https://github.com/onflow/cadence/pull/1002/files#diff-7aeb0f4db759cececb1b38ca7d84c09a8c4ddf85b3ef7e5195f1af04f59b689eL229) to not use the crypto contract.

Reply:
Good idea! I moved the hashing documentation up to the hash algorithm overview in dbfcacc83f48e86c2b623e06c5a755adc1100285. I also fixed some other things